### DISCUSSION TITLE
`output.clean` and `webpack --config-name`

### DISCUSSION THREAD

**Discussion by matzeeable** [webpack/webpack#13551] [Q&A] [answered]
Hey!

I want to use [`output.clean`](https://webpack.js.org/configuration/output/#outputclean). It works perfectly until I try to run a single config within my multi-config. In my app I have multiple configurations, so while developing I use [`$ webpack --config-name`](https://webpack.js.org/api/cli/#config-name). Unfortunately, it also deletes all other chunks from other configs.

Is there a way to keep files of other configurations?

Matthew 😊

**Comment by alexander-akait** [comment, marked as answer]
Keeping them in different directories, previously build don't know about previously files
